utilities/cassandra/merge_operator.h

```diff
@@ -30,8 +32,17 @@ class CassandraValueMergeOperator : public MergeOperator {
 
  virtual bool AllowSingleOperand() const override { return true; }
 
+ virtual bool ShouldMerge(const std::vector<Slice>& operands) const override {
+   if (operands.size() > 0 && operands_limit_ > 0 &&
```

Comment:
if (operands_limit_ > 0 && operands.size() >= operands_limit_) should be sufficient?

Reply:
I remember adding operands.size() > 0 for a specific case, but can't it remember now. 
But, yeah, the condition you mention should be sufficient. I'll update.